docs/about/whats-new.ipynb

```diff
@@ -24,11 +24,106 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "tags": []
+   },
    "source": [
     "## General"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### String formatting\n",
+    "\n",
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "**New in 0.15**\n",
+    "\n",
+    "Added support for compact formatting of 0-D variables.\n",
+    "\n",
+    "</div>\n",
+    "\n",
+    "Example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "var = sc.scalar(12.5, variance=4.0, unit='mm')\n",
+    "print(f'{var}')\n",
+    "print(f'{var:c}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Implicit conversion to boolean\n",
+    "\n",
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "**New in 0.15**\n",
+    "\n",
+    "Added support implicit conversion of 0-D variables to bool.\n",
+    "\n",
+    "</div>\n",
+    "\n",
+    "Examples:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "var = sc.array(values=[1,2,3,4,5], dims=['x'], unit='m')\n",
+    "\n",
+    "if sc.scalar(1, unit='m') < sc.scalar(2, unit='m'):\n",
+    "    print('ok')\n",
+    "\n",
+    "if sc.any(var == sc.scalar(3, unit='m')):\n",
+    "    print('ok')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Keyword-argument syntax for `rename_dims`\n",
+    "\n",
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "**New in 0.15**\n",
+    "\n",
+    "Added support for keyword arguments in `rename_dims` to define dimensions, as also supported by `rename`.\n",
```

Comment:
Should we change the args to match `transform_coords` in this release? Would allow us to avoid mentioning this keyword arg change twice.

Reply:
According to the poll result we should not change it?